tests/release_scraper/test_release_scraper.py

```diff
@@ -32,6 +32,12 @@
     ({'platform': 'win32', 'version': '23.0.1'},
      'firefox-23.0.1.en-US.win32.exe',
      'firefox/releases/23.0.1/win32/en-US/Firefox Setup 23.0.1.exe'),
+    ({'platform': 'win32', 'version': '23'},
```

Comment:
Could you please also add a `23.1.0`release? That way we can test that using `23` as version will skip `23.0.1` and loads `23.1.0` instead.